docker/files/development/etc/nginx/conf.d/default.conf

```diff
@@ -17,6 +17,25 @@ server {
         proxy_set_header X-Amz-Date $authDate;
         proxy_set_header X-Amz-Content-SHA256 $authContentSha256;
 
+        # Get resource from Minio
+        proxy_pass http://minio:9000/drive-media-storage/;
+        proxy_set_header Host minio:9000;
+        add_header Content-Disposition "attachment";
```

Comment:
Luckily we expose the filename in the url, so we don't have to specify filename here 😊